src/main/resources/reference.conf

```diff
@@ -386,7 +386,7 @@ target-validation {
   output {
     succeeded {
       format = ${common.output-format}
-      path = ${common.output}"/targetValidation"
+      path = ${common.output}
     }
     failed {
```

Comment:
@JarrodBaker looking at the current datasets we produce and in the spite of being consistent root level and unmatched sounds better, isn't it?

![image](https://user-images.githubusercontent.com/5394232/135851879-ac0c7d24-2fc1-452f-9305-a09540fa8272.png)

Reply:
@mkarmona we're not going to win any awards as a team for nomenclature, but in general 'Unmatched' is more specific than 'Failed'.